index.html

```diff
@@ -247,16 +247,16 @@ <h3>Background Player</h3>
                 <div class="col-md-9 feature-detail" id="popup" >
                     <h3>Popup Player</h3>
                     <div class="feature-media">
-                        <img src="/img/screenshots/shot_05.png" class="img-responsive" alt="popup player in browser app" />
+                        <img src="/img/screenshots/shot_05.png" class="img-responsive" alt="popup player in web-browser app" />
                     </div>
                     <div class="feature-description">
                         <p><strong>Never stop playing!</strong></p>
                         <ul>
                             <li>View videos whenever you like</li>
                             <li>Resize and move it wherever you like</li>
-                            <li>Use the full comfort of our normal video player</li>
-                            <li class="text-success">Pro tip: switch between all players with one click</li>
-                            <li>Available for all Android versions</li>
+                            <li>In full comfort of our normal video player</li>
+                            <li class="text-success">Pro tip: Switch between all players with one click</li>
```

Comment:
Also should be small letter after colon.

Reply:
It is a full sentence, so can use big versal.

Reply:
This is actually fine in American English, but not in British English. Interesting.

Reply:
Also avoids replications into languages where it is also an error. :)